### ISSUE TITLE
410 Gone - Client Error

### ISSUE THREAD

**Issue by akhiljalagam** [RealOrangeOne/rofi-todoist#4]
After installing it on my void linux machine, I am receiving a 410 Gone - Client Error when trying to add a task.

**Comment by akhiljalagam** [comment]
Now I am getting 403 Forbidden - Client Error 😔

**Comment by RealOrangeOne** [comment]
Do you definitely have a valid authentication token? I've not used `rofi-todoist` in a while

**Comment by akhiljalagam** [comment]
Certainly! I have a token that functions effectively alongside the alternative tool I'm utilizing, which can be found at the following URL: https://github.com/sachaos/todoist.
